<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout">
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QLabel" name="label">
                                        <property name="font">
                                            <font>
                                                <family>Times New Roman</family>
                                                <pointsize>14</pointsize>
                                            </font>
                                        </property>
                                        <property name="text">
                                            <string>Î¼</string>
                                        </property>
                                        <property name="textFormat">
                                            <enum>Qt::AutoText</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QSlider" name="sldMu">
                                        <property name="minimum">
                                            <number>0</number>
                                        </property>
                                        <property name="maximum">
                                            <number>60</number>
                                        </property>
                                        <property name="orientation">
                                            <enum>Qt::Horizontal</enum>
                                        </property>
                                    </widget>
                                </item>
                                <item>
                                    <widget class="QLabel" name="lblEquation">
                                        <property name="sizePolicy">
                                            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                                                <horstretch>0</horstretch>
                                                <verstretch>0</verstretch>
                                            </sizepolicy>
                                        </property>
                                        <property name="minimumSize">
                                            <size>
                                                <width>211</width>
                                                <height>49</height>
                                            </size>
                                        </property>
                                        <property name="maximumSize">
                                            <size>
                                                <width>211</width>
                                                <height>49</height>
                                            </size>
                                        </property>
                                        <property name="text">
                                            <string/>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                        <item>
                            <widget class="PlotWidget" name="vdpPlot"/>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>21</height>
                </rect>
            </property>
            <widget class="QMenu" name="menuPlot">
                <property name="title">
                    <string>Plot</string>
                </property>
                <addaction name="actionTrajectory_plot"/>
                <addaction name="actionPhase_plot"/>
            </widget>
            <widget class="QMenu" name="menuExport">
                <property name="title">
                    <string>Export</string>
                </property>
                <addaction name="actionExport_plot_to_png"/>
            </widget>
            <addaction name="menuPlot"/>
            <addaction name="menuExport"/>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
        <action name="actionTrajectory_plot">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="checked">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>Traectory</string>
            </property>
        </action>
        <action name="actionPhase_plot">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="text">
                <string>Phase plot</string>
            </property>
        </action>
        <action name="actionExport_plot_to_png">
            <property name="text">
                <string>Export plot to png</string>
            </property>
        </action>
    </widget>
    <customwidgets>
        <customwidget>
            <class>PlotWidget</class>
            <extends>QGraphicsView</extends>
            <header>pyqtgraph</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections>
        <connection>
            <sender>actionTrajectory_plot</sender>
            <signal>triggered()</signal>
            <receiver>MainWindow</receiver>
            <slot>trajectoryplot_click()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>-1</x>
                    <y>-1</y>
                </hint>
                <hint type="destinationlabel">
                    <x>399</x>
                    <y>299</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>actionPhase_plot</sender>
            <signal>triggered()</signal>
            <receiver>MainWindow</receiver>
            <slot>phaseplot_click()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>-1</x>
                    <y>-1</y>
                </hint>
                <hint type="destinationlabel">
                    <x>399</x>
                    <y>299</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>sldMu</sender>
            <signal>valueChanged(int)</signal>
            <receiver>MainWindow</receiver>
            <slot>update_mu()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>299</x>
                    <y>55</y>
                </hint>
                <hint type="destinationlabel">
                    <x>399</x>
                    <y>299</y>
                </hint>
            </hints>
        </connection>
        <connection>
            <sender>actionExport_plot_to_png</sender>
            <signal>triggered()</signal>
            <receiver>MainWindow</receiver>
            <slot>export()</slot>
            <hints>
                <hint type="sourcelabel">
                    <x>-1</x>
                    <y>-1</y>
                </hint>
                <hint type="destinationlabel">
                    <x>399</x>
                    <y>299</y>
                </hint>
            </hints>
        </connection>
    </connections>
    <slots>
        <slot>trajectoryplot_click()</slot>
        <slot>phaseplot_click()</slot>
        <slot>update_mu()</slot>
        <slot>export()</slot>
    </slots>
</ui>
